cmd/runcommand.go

```diff
@@ -0,0 +1,34 @@
+package cmd
```

Comment:
The error is caused by cluster_lab.go (https://github.com/hypriot/device-init/blob/master/cmd/cluster_lab.go#L54)

A check for nil or empty string should be added, I guess. Should I fix?